cmd/sonobuoy/app/gen.go

```diff
@@ -88,14 +95,16 @@ func GenFlagSet(cfg *genFlags, rbac RBACMode) *pflag.FlagSet {
 	AddSonobuoyImage(&cfg.sonobuoyImage, genset)
 	AddKubeConformanceImage(&cfg.kubeConformanceImage, genset)
 	AddSystemdLogsImage(&cfg.systemdLogsImage, genset)
-	AddKubeConformanceImageVersion(&cfg.kubeConformanceImageVersion, genset)
 	AddSSHKeyPathFlag(&cfg.sshKeyPath, genset)
 	AddSSHUserFlag(&cfg.sshUser, genset)
 
 	AddPluginSetFlag(&cfg.plugins, genset)
 	AddPluginEnvFlag(&cfg.pluginEnvs, genset)
 
 	AddNodeSelectorsFlag(&cfg.nodeSelectors, genset)
+
+	AddKubeConformanceImageVersion(&cfg.k8sVersion, genset)
```

Comment:
Binding bot to the same variable; not sure if that will behave correctly. Need to make sure that either sets the value.

Reply:
Confirmed either works; if both set the latest applies. Not going to test for that though since it is obviously not desirable behavior to persist.